File: android/versioned-abis/expoview-abi38_0_0/src/main/java/abi38_0_0/expo/modules/camera/tasks/BarCodeScannerAsyncTaskDelegate.java
package abi38_0_0.expo.modules.camera.tasks;

import abi38_0_0.org.unimodules.interfaces.barcodescanner.BarCodeScannerResult;

public interface BarCodeScannerAsyncTaskDelegate {
  void onBarCodeScanned(BarCodeScannerResult barCode);
  void onBarCodeScanningTaskCompleted();
}

File: android/versioned-abis/expoview-abi49_0_0/src/main/java/abi49_0_0/expo/modules/camera/tasks/BarCodeScannerAsyncTaskDelegate.kt
package abi49_0_0.expo.modules.camera.tasks

import abi49_0_0.expo.modules.interfaces.barcodescanner.BarCodeScannerResult

interface BarCodeScannerAsyncTaskDelegate {
  fun onBarCodeScanned(barCode: BarCodeScannerResult)
  fun onBarCodeScanningTaskCompleted()
}
